<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetIcon</class>
 <widget class="QWidget" name="WidgetIcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetPicture" name="widgetPicture" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WidgetShowPicture" name="widgetShowPicture" native="true">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetShowPicture</class>
   <extends>QWidget</extends>
   <header>widgetshowpicture.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
